tests/always/test_project_structure.py

```diff
@@ -420,6 +420,64 @@ def test_missing_assets(self):
         super().test_missing_assets()
 
 
+class TestAmazonProviderProjectStructure(ExampleCoverageTest):
+    PROVIDER = "amazon"
+    CLASS_DIRS = ProjectStructureTest.CLASS_DIRS
```

Comment:
It could be omitted (it will be inherited from ProjectStructureTest). Though now it's more explicit what class dirs you're using so I don't have strong feelings on it.

Reply:
Hey! I'm happy either way, this felt a bit more explicit.